docs/ossm/create-mesh/README.md

```diff
@@ -0,0 +1,121 @@
+# Scoping the service mesh with DiscoverSelectors
+This page describes how the service mesh control plane monitors cluster resources and how to manage its scope. 
```

Comment:
"service mesh control plane" sounds a lot like `SMCP`. Is there something else we want to call this so as not to confuse people?

Reply:
Hmm...hadn't thought of that. It may be hard to get away from at times without spelling out the entire product name "The Red Hat OpenShift Service Mesh control plane..." seems too much. 

This could possibly be simplified to "By default, a service mesh is setup to span the entire cluster. This page describes how to reduce and manage its scope."

Reply:
I don't know how much technical this should be. I think it's important to mention that it's the control plane (istiod) who monitors/discovers the resources. Talking just about "Service mesh" seems to be too general.